test/babel/fixtures/class-properties/block-body/expected.js

```diff
@@ -0,0 +1,21 @@
+var _this = this;
+
+class Foo {
+  bar = (...params) => _this.__bar__REACT_HOT_LOADER__(...params);
+
+  __bar__REACT_HOT_LOADER__(a, b) {
+    return a(b);
+  }
+
+}
+;
```

Comment:
I assumed it was Babel weirdness, I was making the tests pass here. According to [AST explorer (highlight the semicolon)](https://astexplorer.net/#/wK0pmqT3vX) that's an additional empty statement, I'll see if I'm accidentally creating one.